--- opus-0.9.2/celt/tools/Makefile.am.orig	2011-02-14 21:09:02.000000000 +0100
+++ opus-0.9.2/celt/tools/Makefile.am	2011-03-11 19:08:02.971416294 +0100
@@ -15,7 +15,7 @@
 bin_PROGRAMS = celtenc celtdec
 
 celtenc_SOURCES = celtenc.c wav_io.c skeleton.c
-celtenc_LDADD = $(top_builddir)/libcelt/libcelt@LIBCELT_SUFFIX@.la $(OGG_LIBS)
+celtenc_LDADD = $(top_builddir)/libcelt/libopuscelt@LIBCELT_SUFFIX@.la $(OGG_LIBS)
 
 celtdec_SOURCES = celtdec.c wav_io.c
-celtdec_LDADD = $(top_builddir)/libcelt/libcelt@LIBCELT_SUFFIX@.la $(OGG_LIBS)
+celtdec_LDADD = $(top_builddir)/libcelt/libopuscelt@LIBCELT_SUFFIX@.la $(OGG_LIBS)
--- opus-0.9.2/celt/libcelt/Makefile.am.orig	2011-02-28 22:28:54.000000000 +0100
+++ opus-0.9.2/celt/libcelt/Makefile.am	2011-03-11 19:07:39.119417970 +0100
@@ -11,14 +11,14 @@
 
 noinst_SCRIPTS = match-test.sh
 
-lib_LTLIBRARIES = libcelt@LIBCELT_SUFFIX@.la
+lib_LTLIBRARIES = libopuscelt@LIBCELT_SUFFIX@.la
 
 # Sources for compilation in the library
-libcelt@LIBCELT_SUFFIX@_la_SOURCES = bands.c celt.c cwrs.c ecintrin.h entcode.c \
+libopuscelt@LIBCELT_SUFFIX@_la_SOURCES = bands.c celt.c cwrs.c ecintrin.h entcode.c \
 	entdec.c entenc.c header.c kiss_fft.c laplace.c mathops.c mdct.c \
 	modes.c pitch.c plc.c quant_bands.c rate.c vq.c
 
-libcelt@LIBCELT_SUFFIX@_la_LDFLAGS = \
+libopuscelt@LIBCELT_SUFFIX@_la_LDFLAGS = \
 	-version-info @CELT_LT_CURRENT@:@CELT_LT_REVISION@:@CELT_LT_AGE@ \
 	-no-undefined
 
@@ -31,9 +31,9 @@
 
 noinst_PROGRAMS = testcelt dump_modes
 testcelt_SOURCES = testcelt.c
-testcelt_LDADD = libcelt@LIBCELT_SUFFIX@.la
+testcelt_LDADD = libopuscelt@LIBCELT_SUFFIX@.la
 INCLUDES = 
 #libcelt@LIBCELT_SUFFIX@_la_LIBADD =
 
 dump_modes_SOURCES = dump_modes.c
-dump_modes_LDADD = libcelt@LIBCELT_SUFFIX@.la
+dump_modes_LDADD = libopuscelt@LIBCELT_SUFFIX@.la
--- opus-0.9.2/celt/tests/Makefile.am.orig	2011-02-14 21:09:02.000000000 +0100
+++ opus-0.9.2/celt/tests/Makefile.am	2011-03-11 19:28:09.623417970 +0100
@@ -14,4 +14,4 @@
 #rotation_test_SOURCES = rotation-test.c
 mathops_test_SOURCES = mathops-test.c
 tandem_test_SOURCES = tandem-test.c
-tandem_test_LDADD = $(top_builddir)/libcelt/libcelt@LIBCELT_SUFFIX@.la
+tandem_test_LDADD = $(top_builddir)/libcelt/libopuscelt@LIBCELT_SUFFIX@.la
--- opus-0.9.2/silk/Makefile.am.orig	2011-03-11 19:28:47.939416294 +0100
+++ opus-0.9.2/silk/Makefile.am	2011-03-11 19:50:01.543418808 +0100
@@ -215,4 +215,4 @@
 signalCompare_LDADD = $(lib_LTLIBRARIES) -lm
 signalCompare_LDFLAGS = $(LIBS)
 
-LIBS = ../celt/libcelt/libcelt0.la
+LIBS = ../celt/libcelt/libopuscelt0.la
